lib/dynamoid/associations/association.rb

```diff
@@ -58,6 +58,12 @@ def declaration_field_type
         :set
       end
 
+      def disassociate_source
+        Array(target).each do |target_entry|
+          target_entry.send(target_association).disassociate(source.hash_key) if target_entry && target_association
+        end
+      end
```

Comment:
**minor:** As far as it's a new public method - it could be covered with unit tests as well.